docker-compose.yml

```
@@ -60,10 +67,77 @@ services:
+  snak-bff:
+    image: ghcr.io/kasarlabs/snak-app/snak-bff:sha-93469bd
+    ports:
+      - "3001:3001"
+    networks: [snaknet]
+  
+  snak-frontend:
+    image: ghcr.io/kasarlabs/snak-app/snak-frontend:sha-93469bd
+    ports:
+      - "3000:3000"
+    networks: [snaknet]
 
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)